src/components/IntervieweeAddModalView/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import ModalView from "../ModalView";
+
+const Title = styled.div`
+  display: flex;
+  width: 20%;
+  margin: 10px;
+  font-size: 1.5rem;
+  font-weight: 500;
+`;
+
+const EditField = styled.div`
+  display: flex;
+  margin: 10px;
+`;
+
+const Form = styled.form`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 100%;
+  height: 70%;
+  margin: 15px;
+`;
+
+const Label = styled.div`
+  display: flex;
+  justify-content: center;
+  justify-content: flex-start;
+  width: 10%;
+  align-items: center;
+
+  font-size: 1.3rem;
+  font-weight: 450;
+`;
+
+const Input = styled.input`
+  margin: 5px 20px;
+  padding: 3px 10px 0;
+  width: 70%;
+  height: 30px;
+  font-size: 1rem;
+`;
+
+const Button = styled.button`
+  width: 70px;
+  height: 40px;
+  font-size: 1rem;
+  margin: 5px;
+  cursor: pointer;
+`;
+
+const BtnGroup = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 40px;
+`;
+
+export default function IntervieweeAddModalView({ onFormSubmitBtnClick, onCancleBtnClick }) {
+  const [selectedFile, setSelectedFile] = useState(null);
+  const [intervieweeInfo, setIntervieweeInfo] = useState({ name: "", email: "" });
+
+  function handleInputChange(e) {
+    e.preventDefault();
+
+    const intervieweeInputType = e.target.name;
+    const value = e.target.value;
```

Comment:
c
```suggestion
    const { target: { name, value } } = event;
    const intervieweeInputType = event.target.name;
    const value = event.target.value;
```

요런식으로 디스트럭쳐링 해도 좋을 것 같아요!
그리고 저희의 컨벤션은 event 풀 네임이었던 것으로 기억합니다!

Reply:
네 event로 통일하겠습니다!